### DISCUSSION TITLE
Set timeout for GPT model

### DISCUSSION THREAD

**Discussion by szoke120** [langchain4j/langchain4j#237] [Q&A]
Hi!

This library is very useful and easy to use! However, I can't find where timeout can be changed. I found it to be the default 20L but I can't rewrite it.

**Comment by ice09** [comment]
Did you try this?

```
OpenAiChatModel assistant =
        OpenAiChatModel.builder()
                .modelName("gpt-3.5-turbo")
                .timeout(Duration.ofMinutes(5)).build();
```

**Reply by szoke120** [reply]
Hi!

Thanks! 
Finally, I modified this prop in application.properties: langchain4j.chat-model.openai.timeout=PT120S
And it works!
